pkgdb2/lib/__init__.py

```diff
@@ -2025,6 +2025,134 @@ def bugzilla(session, name=None):
     return output
 
 
+def _vcs_acls_json(packages, skip_pp=None):
+    """ For a given list of package/user/branch build a dict of dict
+    representating of who has commit access to which package.
+
+    The output dict is something like:
+
+    {
+      pkg1: {
+        branch1: {
+          name: pkg1,
+          branch: branch1,
+          user: [user1, user2],
+          group: [group1, group2]
```

Comment:
This looks like it should be ``groups`` not ``group`` and ``people`` not ``user`` (from reading the code below).